utils.py

```diff
@@ -423,7 +423,7 @@ def base64_from_int(value):
     Returns:
         *. Returns the base64 representation of the number passed.
     """
-    return base64.b64encode(bytes([value]))
+    return base64.b64encode(python_utils.convert_to_bytes(value))
```

Comment:
One question does this give ± same length of the base64 strings?

Reply:
Done